_examples/mini-habr-with-subscriptions/README.md

```diff
@@ -0,0 +1,272 @@
+`│   ├── model/`<br>
+`│   │   └──` [`model.go`](./internal/model/model.go)                (Internal data models)<br>
+`│   └── storage/`<br>
+`│       ├── db/` (Implementation of database storage) <br>
+`│       │   └──` [`resolvers.go`](./internal/storage/db/resolvers.go)        (Implementation of methods for working with PostgreSQL database)<br>
+`│       ├── in-memory/` (Implementation of in-memory data storage) <br>
+`│       │   └──` [`resolvers.go`](./internal/storage/in-memory/resolvers.go)        (Implementation of methods for working with in-memory data)<br>
+`│       └──` [`interface.go`](./internal/storage/interface.go)            (Interface for data storage (PostgreSQL, in-memory))<br>
+`├── migrations/`<br>
+`│   └──` [`001_create_tables.up.sql`](./migrations/001_create_tables.up.sql)    (SQL script for database migration (creating tables))<br>
+`├── tools/`<br>
+`│    └──` [`tools.go`](./tools/tools.go)                   (Tools for gqlgen code generation)<br>
+`├──` [`.env`](./.env)                            (Environment variables file (database settings, Redis, etc.))<br>
+`├──` [`.gitignore`](./.gitignore)                      (List of ignored files and directories for Git)<br>
+`├──` [`docker-compose.yml`](./docker-compose.yml)              (Docker Compose configuration for launching the application and dependencies)<br>
+`├──` [`Dockerfile`](./Dockerfile)                      (Instructions for building Docker image)<br>
+`├──` [`go.mod`](./go.mod)                          (Go dependencies file)<br>
+`├──` [`go.sum`](./go.sum)                          (Go dependencies checksums file)<br>
+`├──` [`gqlgen.yml`](./gqlgen.yml)                      (Configuration file for gqlgen)<br>
+`├──` [`LICENSE`](./LICENSE)                         (Project license)<br>
+`└──` [`README.md`](./README.md)                       (Project description file)<br>
+
+</details>
+
+
+## 🔧 Stack:
+  * Go 1.24
+  * GraphQL (gqlgen)
+  * PostgreSQL 17
+  * Redis 9
+  * Docker & Docker Compose
+  * Gorilla WebSockets
+
+## 📜 License
+
+This project is distributed under the Apache License 2.0. This is a free open-source license that allows the use, modification, and distribution of code for both commercial and non-commercial purposes.
+
+The full text of the license is available in the [LICENSE](./LICENSE) file.
+
+```
+Copyright 2023 Ozon Habr API
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+```
```

Comment:
You should omit this whole thing as it's covered under the license at the root of the repository.